ps0.py

```diff
@@ -1,10 +1,13 @@
+import math
+
 # 1. Ask the user to enter a number "x"
-
+x = float(input("Enter a value for x: "))
 
 # 2. Ask the user to enter a number "y"
-
+y = float(input("Enter a value for y: "))
 
 # 3. Prints out the number "x" raised to the power "y"
+print("\n" + "\"x\" raised to the power \"y\" = " + str(x ** y) + "\n")
 
-
-# 4. Prints out the log (base 2) of "x"
\ No newline at end of file
+# 4. Prints out the log (base 2) of "x"
+print("Log base 2 of \"x\" = " + str(math.log(x,2)) + "\n")
```

Comment:
There is a `math.log2()` that will return more precise results, but your approach is good too.